core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1284,9 +1269,16 @@ object GroupMetadataManager {
         val leaderId = value.get(LEADER_KEY).asInstanceOf[String]
         val memberMetadataArray = value.getArray(MEMBERS_KEY)
         val initialState = if (memberMetadataArray.isEmpty) Empty else Stable
-        val currentStateTimestamp: Option[Long] =
-          if (version > 1) Some(value.getLong(CURRENT_STATE_TIMESTAMP_KEY))
-          else None
+        val rawCurrentStateTimestamp = value.getLong(CURRENT_STATE_TIMESTAMP_KEY)
```

Comment:
Won't this raise an exception for the older schemas? Do we have any test cases covering this?